dotnet/src/webdriver/WebDriver.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>net6.0;net5.0;net48;net47;net46;net45;netstandard2.1;netstandard2.0</TargetFrameworks>
     <AssemblyName>WebDriver</AssemblyName>
     <RootNamespace>OpenQA.Selenium</RootNamespace>
+    <Configurations>Release;Debug</Configurations>
```

Comment:
Any particular reason to specify the supported configurations per project? I supposed we can even reduce this file, but you adds more lines.

Reply:
My bad, I think the IDE added that by default. I noticed that and I was going to revert the change and then I just forgot.
Reverted through https://github.com/SeleniumHQ/selenium/commit/b9ba020876bfa5222cca30e696e2644ee6fcb580